### ISSUE TITLE
libffi: Cannot initialize AppImage on Ubuntu 20.04

### ISSUE THREAD

**Issue by tferr** [napari/napari#1627]
I get the following exception when starting the napari-0.3.7-x86_64.AppImage on Ubuntu 20.04:

```
(AppImageLauncher:116703): GLib-CRITICAL **: 16:46:26.076: g_file_test: assertion 'filename != NULL' failed
gtk-update-icon-cache: Cache file created successfully.
Traceback (most recent call last):
  File "/tmp/.mount_napariJj0aiL/usr/lib/python3.8/runpy.py", line 185, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/tmp/.mount_napariJj0aiL/usr/lib/python3.8/runpy.py", line 144, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/tmp/.mount_napariJj0aiL/usr/lib/python3.8/runpy.py", line 111, in _get_module_details
    __import__(pkg_name)
  File "/tmp/.mount_napariJj0aiL/usr/app/napari/__init__.py", line 48, in <module>
    from vispy import app
  File "/tmp/.mount_napariJj0aiL/usr/app_packages/vispy/__init__.py", line 30, in <module>
    from .util import config, set_log_level, keys, sys_info  # noqa
  File "/tmp/.mount_napariJj0aiL/usr/app_packages/vispy/util/__init__.py", line 14, in <module>
    from . import fonts       # noqa
  File "/tmp/.mount_napariJj0aiL/usr/app_packages/vispy/util/fonts/__init__.py", line 13, in <module>
    from ._triage import _load_glyph, list_fonts  # noqa, analysis:ignore
  File "/tmp/.mount_napariJj0aiL/usr/app_packages/vispy/util/fonts/_triage.py", line 11, in <module>
    from ._freetype import _load_glyph
  File "/tmp/.mount_napariJj0aiL/usr/app_packages/vispy/util/fonts/_freetype.py", line 16, in <module>
    from ...ext.fontconfig import find_font
  File "/tmp/.mount_napariJj0aiL/usr/app_packages/vispy/ext/fontconfig.py", line 4, in <module>
    from ctypes import (util, cdll, c_void_p, c_char_p, c_double, c_int, c_bool,
  File "/tmp/.mount_napariJj0aiL/usr/lib/python3.8/ctypes/__init__.py", line 7, in <module>
    from _ctypes import Union, Structure, Array
ImportError: libffi.so.6: cannot open shared object file: No such file or directory
```

**Comment by tlambert03** [comment]
thanks for the report @tferr.  I'm afraid that's a known issue without a resolution at the moment.
The issue with `libffi.so.6` appears to be [fixed in briefcase 0.3.3](https://github.com/beeware/briefcase/issues/420#issuecomment-660442534).  However, our build pipeline has been [pinned to version 0.3.1](https://github.com/napari/napari/pull/1599), because there's a different [issue building Pyside2-dependent apps (like ours) with briefcase 0.3.3](https://github.com/beeware/briefcase/issues/470).

for linux 20.04, looks like you'll have to install via conda or pip for now until those bugs are ironed out.

**Comment by jaimergp** [comment]
This shouldn't be an issue anymore with the [conda-based installers available for 0.4.15](https://github.com/napari/napari/releases/tag/v0.4.15) and above. For Linux, check [napari-0.4.15-Linux-x86_64.sh](https://github.com/napari/napari/releases/download/v0.4.15/napari-0.4.15-Linux-x86_64.sh)! Once downloaded, you can start the installation with `bash ~/Downloads/napari-0.4.15-Linux-x86_64.sh` (adjust the location to your system if necessary).

I think we can close this one @potating-potato

**Comment by liu-ziyang** [comment]
For now I am going to close this issue as we have alternative app available for the latest build of napari. If someone else encounter similar problem with the latest bundle or have more information on this issue, please feel free to reopen it
